src/mplex.ts

```diff
@@ -41,7 +41,7 @@ function printMessage (msg: Message) {
 }
 
 export interface MplexStream extends Stream {
-  source: Pushable<Uint8Array>
```

Comment:
Remind me why the rename is necessary?

`Stream` extends `Duplex` from `it-stream-types` which defines the `source` property, so now MplexStream instances will have both `source` and `inputSource` properties.

Reply:
happy to find another way of doing https://github.com/libp2p/js-libp2p/pull/1290
Problem with current upstream approach is that `{...stream}` only graps enumerable properties, misses methods (like `close`). Since mplex stream isn't a class, its not effected.